README.md

```diff
@@ -201,9 +203,14 @@ MESH configuration is done using environment variables:
 | NHAIS_MESH_ENDPOINT_CERT         |                           | The content of the PEM-formatted client endpoint certificate
 | NHAIS_MESH_ENDPOINT_PRIVATE_KEY  |                           | The content of the PEM-formatted client private key
 | NHAIS_MESH_SUB_CA                |                           | The content of the PEM-formatted certificate of the issuing Sub CA. Empty if NHAIS_MESH_CERT_VALIDATION is false
-| NHAIS_MESH_CYPHER_TO_MAILBOX     |                           | The mapping between each HA Trading Partner Code (Link Code) to its corresponding MESH Mailbox ID mapping. There is one mapping per lines and an equals sign (=) separates the code and mailbox id. For example: "COD1=A6840385\nHA01=A0047392"
+| NHAIS_MESH_CYPHER_TO_MAILBOX     |                           | (1) The mapping between each HA Trading Partner Code (HA Link Code) to its corresponding MESH Mailbox ID mapping. There is one mapping per lines and an equals sign (=) separates the code and mailbox id. For example: "COD1=A6840385\nHA01=A0047392"
```

Comment:
We should consider renaming this variable to something like `NHAIS_MESH_LINK_CODES_TO_MAILBOX_IDS`